plugins/rule_engines/irods_rule_language/include/irods/private/re/utils.hpp

```diff
@@ -104,38 +104,38 @@ int isRuleGenSyntax( char *expr );
 #define CONCAT(a,b) CONCAT2(a,b)
 
 #define KEY_PROTO
-#include "key.proto.hpp"
-#include "proto.hpp"
-#include "restruct.templates.hpp"
-#include "end.instance.hpp"
+#include "irods/private/re/key.proto.hpp"
+#include "irods/private/re/proto.hpp"
+#include "irods/private/re/restruct.templates.hpp"
+#include "irods/private/re/end.instance.hpp"
 
 void keyBuf( unsigned char *buf, int size, char *keyBuf );
 
 #undef KEY_PROTO
 
-#include "region.to.region.proto.hpp"
-#include "proto.hpp"
-#include "restruct.templates.hpp"
-#include "end.instance.hpp"
-
-#include "to.region.proto.hpp"
-#include "proto.hpp"
-#include "restruct.templates.hpp"
-#include "end.instance.hpp"
-
-#include "to.memory.proto.hpp"
-#include "proto.hpp"
-#include "restruct.templates.hpp"
-#include "end.instance.hpp"
-
-#include "region.to.region2.proto.hpp"
-#include "proto.hpp"
-#include "restruct.templates.hpp"
-#include "end.instance.hpp"
-
-/*#include "region.check.proto.hpp"
-#include "restruct.templates.hpp"
-#include "end.instance.hpp"*/
+#include "irods/private/re/region.to.region.proto.hpp"
+#include "irods/private/re/proto.hpp"
+#include "irods/private/re/restruct.templates.hpp"
+#include "irods/private/re/end.instance.hpp"
+
+#include "irods/private/re/to.region.proto.hpp"
+#include "irods/private/re/proto.hpp"
+#include "irods/private/re/restruct.templates.hpp"
+#include "irods/private/re/end.instance.hpp"
+
+#include "irods/private/re/to.memory.proto.hpp"
+#include "irods/private/re/proto.hpp"
+#include "irods/private/re/restruct.templates.hpp"
+#include "irods/private/re/end.instance.hpp"
+
+#include "irods/private/re/region.to.region2.proto.hpp"
+#include "irods/private/re/proto.hpp"
+#include "irods/private/re/restruct.templates.hpp"
+#include "irods/private/re/end.instance.hpp"
+
+/*#include "irods/private/re/region.check.proto.hpp"
+#include "irods/private/re/restruct.templates.hpp"
+#include "irods/private/re/end.instance.hpp"*/
```

Comment:
anyone have thoughts on this?   clearly can be done after this PR - but... seems suspect.

Reply:
I think it might be worth filing an issue to understand `end.instance.hpp`.

Reply:
looks like a general purpose macro cleaner.

Reply:
i'm referring to the `end.instance.hpp` file

Reply:
@trel, what are you referring to specifically? Just the commented-out bits? Or the whole framework that was built here?

Reply:
the whole thing - why multiple includes of identical things - better way to approach it - and then.. the commented out code.    really, just feels ripe for a revisit.   yes, please create an issue for investigation/refactor.

Reply:
related: #6210

Reply:
I think that'll do it. @trel, good?